packages/metascraper-helpers/index.js

```diff
@@ -197,12 +197,8 @@ const date = value => {
   // convert isodates to restringify, because sometimes they are truncated
   if (isIso(value)) return new Date(value).toISOString()
 
-  // try to parse with the built-in date parser
-  const native = new Date(value)
-  if (!isNaN(native.getTime())) return native.toISOString()
-
   // try to parse a complex date string
-  const parsed = chrono.parseDate(value)
+  const parsed = chrono.parseDate(value, new Date(''), { forwardDate: true }) // magick
```

Comment:
@slavaGanzin can you explain why these additional parameters are necessary? I don't understand it 🙂

Reply:
mainly because

```
> new Date('')
Invalid Date
````

Reply:
@Kikobeats I'm not sure why it works, but it was the only way I can get from chrono to convert any date without year to current year. Because in publishing if year is not specified it means "this year". TechCrunch and Substack, for example, do so

But chrono make some bogus assumptions:
```js
SyntaxError: Unexpected identifier
> require('chrono-node').parseDate('Jan 20')
2020-01-20T10:00:00.000Z
> require('chrono-node').parseDate('May 20')
2020-05-20T09:00:00.000Z
> require('chrono-node').parseDate('Sep 20')
2019-09-20T09:00:00.000Z
> require('chrono-node').parseDate('Dec 20')
2019-12-20T10:00:00.000Z
```

While explaining issue I found better way:
```
> require('chrono-node').parseDate('Jan 10', String(new Date().getFullYear()), {forwardDate: true})
```

So this should be 'default', add this please